src/main/java/com/google/devtools/build/lib/sandbox/DummySandboxedSpawn.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 The Bazel Authors. All rights reserved.
```

Comment:
This class is used for Windows only.
Could you please rename to WindowsSandboxedSpawn or something other with Windows in the name?